lib/shopify_app/sessions.rb

```diff
@@ -1,5 +1,5 @@
 module ShopifyApp
-  module SessionsController
```

Comment:
I'll keep it out of this PR but given that v7 already has a bunch of name changes I'll put up another PR to rename LoginProtection to AuthConcern to keep things a bit consistent